projectile.el

```diff
@@ -358,6 +358,12 @@ containing a root file."
   :group 'projectile
   :type '(repeat function))
 
+(defcustom projectile-dirconfig-comment-prefix
+  nil
+  "If specified, starting a line in a project's .projectile file with this character marks that line as a comment instead of a pattern. Similar to '#' in .gitignore files."
+  :group 'projectile
+  :type 'character)
```

Comment:
Don't forget to add the package-version property here.

Reply:
done!